management/src/test/java/org/kaazing/gateway/management/jmx/JmxRoundTripLatencyIT.java

```diff
@@ -93,7 +93,6 @@
                             .realm()
                                 .name("jmxrealm")
                                 .description("realm for jmx")
```

Comment:
I believe these were removed because the tests don't execute any actual challenging/token.  Please confirm.  (Also I'm unclear why login modules are needed at all for this test, maybe round trip latency requires them?)